pkg/lib/cockpit-components-terminal.tsx

```diff
@@ -284,42 +400,13 @@ export class Terminal extends React.Component {
                     .catch(e => console.error('Text could not be copied, use Ctrl+Insert ', e ? e.toString() : ""))
                     .finally(() => this.terminal.focus());
         } catch (error) {
-            console.error('Text could not be copied, use Ctrl+Insert:', error.toString());
-        }
-    }
-
-    onChannelMessage(event, data) {
-        this.terminal.write(data);
-    }
-
-    onChannelClose(event, options) {
-        const term = this.terminal;
-        term.write('\x1b[31m' + (options.problem || 'disconnected') + '\x1b[m\r\n');
-        term.cursorHidden = true;
-        term.refresh(term.rows, term.rows);
-    }
-
-    connectChannel() {
-        const channel = this.props.channel;
-        if (channel?.valid) {
-            channel.addEventListener('message', this.onChannelMessage.bind(this));
-            channel.addEventListener('close', this.onChannelClose.bind(this));
+            console.error('Text could not be copied, use Ctrl+Insert:', String(error));
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21680-b536158f-20250901-105310-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)